Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -43,17 +40,12 @@ class Navigator {
             
             self?.navigator.setRoutesFor(routesParams) { [weak self] result in
                 if result.isValue() {
-                    os_log("Navigator has been updated",
-                           log: Navigator.log,
-                           type: .debug)
+                    Log.info("Navigator has been updated", category: .navigation)
```

Comment:
Was it intended to change log level here?
```suggestion
                    Log.debug("Navigator has been updated", category: .navigation)
```